MOBILE_API.xml

```diff
@@ -6606,7 +7077,118 @@
+
+        <param name="appServiceManifest" type="AppServiceManifest" mandatory="true">
+            <description> The manifest of the service that wishes to be published.</description>
+        </param>
+    </function>
+
+    <function name="PublishAppService" functionID="PublishAppServiceID" messagetype="response" since="5.1">
+        <description>Response to the request to register a service offered by this app on the module</description>
+
+        <param name="success" type="Boolean" platform="documentation" mandatory="true">
+            <description> true, if successful; false, if failed </description>
+        </param>
+       
+        <param name="resultCode" type="Result" platform="documentation" mandatory="true">
+            <description>See Result</description>
+            <element name="SUCCESS"/>
+            <element name="REJECTED"/>
+            <element name="DISALLOWED"/>
+            <element name="INVALID_DATA"/>
+            <element name="OUT_OF_MEMORY"/>
+            <element name="TOO_MANY_PENDING_REQUESTS"/>
+            <element name="APPLICATION_NOT_REGISTERED"/>
+            <element name="GENERIC_ERROR"/>
+        </param>
+
+        <param name="info" type="String" maxlength="1000" mandatory="false" platform="documentation">
+            <description>Provides additional human readable info regarding the result.</description>
+        </param>
+
+        <param name="appServiceRecord" type="AppServiceRecord" mandatory="false">
+            <description> If the request was successful, this object will be the current status of the service record for the published service. This will include the Core supplied service ID.</description>
+        </param>
+    </function>
+
+    <function name="GetAppServiceData" functionID="GetAppServiceDataID" messagetype="request" since="5.1">
+        <description> This request asks the module for current data related to the specific service. It also includes an option to subscribe to that service for future updates</description>
+        
+        <param name="serviceType" type="String" mandatory="true">
+		    <description>The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core.</description>
+	    </param>
+        
+        <param name="subscribe" type="Boolean" mandatory="false">
+            <description> If true, the consumer is requesting to subscribe to all future updates from the service publisher. If false, the consumer doesn't wish to subscribe and should be unsubscribed if it was previously subscribed.</description>
+        </param>
+    </function>
+
+    <function name="GetAppServiceData" functionID="GetAppServiceDataID" messagetype="response" since="5.1">
+        <description> This response includes the data that is requested from the specific service</description>
```

Comment:
_This response includes the data that is requested from the specific service_
should be 
_This response includes the data that was requested from the specific service_

Reply:
Fixed in https://github.com/smartdevicelink/rpc_spec/pull/149/commits/06a3394c636aa24d9b880a352c67c94700476e0e